sdk/storage/storage-blob-changefeed/review/storage-blob-change-feed.api.md

```diff
@@ -0,0 +1,91 @@
+## API Report File for "@azure/storage-blob-changefeed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
+export class BlobChangeFeedClient {
+    constructor(blobServiceClient: BlobServiceClient);
+    // (undocumented)
+    getChanges(options?: ChangeFeedGetChangesOptions): PagedAsyncIterableIterator<BlobChangeFeedEvent, BlobChangeFeedEventPage>;
+    }
+
+// @public (undocumented)
+export interface BlobChangeFeedEvent {
+    // (undocumented)
+    data: BlobChangeFeedEventData;
+    // (undocumented)
+    dataVersion?: string;
+    // (undocumented)
+    eventTime: string;
+    // (undocumented)
+    eventType: BlobChangeFeedEventType;
+    // (undocumented)
+    id: string;
+    // (undocumented)
+    metadataVersion: string;
+    // (undocumented)
+    subject: string;
+    // (undocumented)
+    topic: string;
+}
+
+// @public (undocumented)
+export interface BlobChangeFeedEventData {
+    // (undocumented)
+    api: string;
+    // (undocumented)
+    blobType: BlobType;
+    // (undocumented)
+    clientRequestId: string;
+    // (undocumented)
+    contentLength: number;
+    // (undocumented)
+    contentType: string;
+    // (undocumented)
+    destinationUrl?: string;
+    // (undocumented)
+    eTag: string;
```

Comment:
`etag` for consistency with other storage libraries?

Reply:
Which language are you referring to?
.Net has
```
public ETag ETag { get; internal set; }
```

Reply:
The JS storage libraries.